src/ansys/fluent/core/utils/search.py

```diff
@@ -21,12 +33,36 @@
     Workflow,
 )
 
+_THIS_DIRNAME = os.path.dirname(__file__)
+
+
+class ValueConflict(ValueError):
```

Comment:
It's better to throw this one directly as a `ValueError`.

The message can be something like "``wildcard`` cannot be ``True`` when ``match_whole_word`` is ``True``."

Reply:
Done. Thank you.